content-src/experiments/containers.yaml

```diff
@@ -9,16 +9,17 @@ incompatible:
   'blok@mozilla.org': 'Tracking Protection'
 thumbnail: /static/images/experiments/containers/icon/thumbnail.png
 description: >
-  Contain yourselves! Create different containers for each of your online lives -  
+  Contain yourselves! Create different containers for each of your online lives -
   your work self, your social self, your personal self - to stay organized and maintain privacy.
+warning: >
```

Comment:
@johngruen - Need an r+ here, since it looks like [you were the last to touch the `introduction` here](https://github.com/mozilla/testpilot/blame/master/content-src/experiments/containers.yaml#L15). The HTML here is no bueno for L10n